server/routes/teams.ts

```diff
@@ -77,6 +82,7 @@ export default function routes({
 
       const displayTeam = {
         name: team.name,
+        component: componentList,
```

Comment:
Shouldn't `component` here also be called `componentList` as it's not always single component?